cockatrice/src/userinfobox.cpp

```diff
@@ -110,6 +110,11 @@ void UserInfoBox::updateInfo(const ServerInfo_User &user)
         userLevelText = tr("Registered user");
     else
         userLevelText = tr("Unregistered user");
+
+    if (user.has_privlevel() && user.privlevel() != "NONE") {
+        userLevelText += tr(" | %1").arg(user.privlevel().c_str());
```

Comment:
Can you exclude " | " from tr() please? Actually there is nothing to translate at all here... ;P